### DISCUSSION TITLE
LookDown: Ability to search/filter dropdown values

### DISCUSSION THREAD

**Discussion by wenstarner** [khoait/DCE.PCF#145] [Ideas] [answered]
Hello again, just another idea that came to mind when using LookDown...

I was wondering if you might think it a good idea to implement the ability to search/filter dropdown values?

Using the UK counties as an example in your documentation, users might find it really helpful to quickly type in a few letters to narrow down to the value they want.

Here's an example of how it looks with the OOB component, which nicely highly highlights in bold the match against values, based on search input:

![image](https://github.com/user-attachments/assets/1a487a25-fbec-4215-a243-5a5a7ba4ffb1)

Again, just an idea that you might wish to consider!

Thanks again :)

**Comment by khoait** [comment, marked as answer]
@wenstarner thanks for your idea! the target use case of lookdown is to simplify a lookup field by allowing users to quickly pick an option from a dropdown without searching or pressing enter, making it feel like a Choice field. adding search feature will turn it back to a normal lookup field.

**Reply by wenstarner** [reply]
Fair enough… I have a long list and prefer your component because it is easier on the eyes than the OOB lookup, especially with the ability to group options. 

But you make a good point… As always, thanks for considering and responding! Very much appreciated 😊

**Reply by khoait** [reply]
could be an idea to start a new advanced lookup field :)
